geomesa-gs-plugin/geomesa-accumulo-gs-plugin/README.md

```diff
@@ -1,11 +1,33 @@
-# GeoMesa Plugin
+# GeoMesa Accumulo GeoServer Plugin
 
-### Building Instructions
+### Installation Instructions
```

Comment:
http://www.geomesa.org/geomesa-deployment/ some other things listed there is information about JAI, vecmath, jline. I doubt jline is needed for the plugin but not sure if vecmath will help